install/kubernetes/cilium/templates/cilium-agent-clusterrolebinding.yaml

```diff
@@ -9,6 +9,6 @@ roleRef:
   name: cilium
 subjects:
 - kind: ServiceAccount
-  name: cilium
+  name: {{ .Values.serviceAccounts.cilium.name | quote }}
```

Comment:
why the `quote`? none of the others have quote so we should don't quote this one as well. Or quote all other others, for consistency.

Reply:
@aanm this is coming from [here](https://github.com/cilium/cilium/pull/14711#discussion_r563636978
)

Reply:
Let's put all remaining service account names quoted as well.

Reply:
Done.